chat/models.py

```diff
@@ -0,0 +1,21 @@
+from django.db import models
+from rest_framework.pagination import PageNumberPagination
+
+
+from targets.models import Match
+from users.models import CustomUser
+
+
+class MatchMessage(models.Model):
+    in_match = models.ForeignKey(
+        Match, on_delete=models.CASCADE, related_name='messages')
+    content = models.CharField(max_length=350)
+    created_at = models.DateTimeField(auto_now_add=True)
+    sent_by = models.ForeignKey(
+        CustomUser, on_delete=models.CASCADE, related_name='creator')
```

Comment:
```suggestion
       sent_by = models.ForeignKey(CustomUser, 
                                   on_delete=models.CASCADE, 
                                   related_name='creator')
```
Don't you think the code looks nicer this way?

Reply:
Yes, I will change it